definitions/private-law/json/ComplexTypes/UploadAdditionalApplications/2_C2DocumentBundle/ComplexTypes.json

```diff
@@ -11,167 +11,163 @@
   {
     "LiveFrom": "20/11/2022",
     "ID": "C2DocumentBundle",
-    "ListElementCode": "document",
-    "FieldType": "Document",
-    "ElementLabel": "File",
-    "RegularExpression": ".doc,.docx,.pdf",
+    "ListElementCode": "author",
```

Comment:
will it have any issues for older cases?

Reply:
no as this event was never enabled in prod.